<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaesarDialog</class>
 <widget class="QDialog" name="CaesarDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>105</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Caesar Properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="8" column="1">
    <widget class="QSpinBox" name="ShiftSpinBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximum">
      <number>8192</number>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="ShiftLabel">
     <property name="text">
      <string>Shift:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="AlphabetLabel">
     <property name="text">
      <string>Alphabet:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="AlphabetLineEdit"/>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="EncryptPushButton">
     <property name="text">
      <string>Encrypt</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>AlphabetLineEdit</tabstop>
  <tabstop>ShiftSpinBox</tabstop>
  <tabstop>EncryptPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
